src/api/app/models/project.rb

```diff
@@ -491,10 +491,10 @@ def can_be_unlocked?(with_exception = true)
       maintenance_release_requests = requests.where(bs_request_actions: { type: 'maintenance_release', source_project: self.name})
       if maintenance_release_requests.exists?
         if with_exception
-          raise OpenReleaseRequest.new "Unlock of maintenance incident #{self.maintenance_incident.project.name} is not possible," +
+          raise OpenReleaseRequest.new "Unlock of maintenance incident #{self.name} is not possible," +
                                        " because there is a running release request: #{maintenance_release_requests.first.id}"
         else
-          errors.add(:base, "Unlock of maintenance incident #{self.maintenance_incident.project.name} is not possible," +
```

Comment:
@adrianschroeter Ana figured out that the maintenance incident points back to the project. So probably an association we could remove from the MaintenanceIncident model.